source/sdk/structs/Stack.ooc

```diff
@@ -1,73 +1,54 @@
-import structs/ArrayList
-
-Stack: class <T> extends BackIterable<T> {
-    data: ArrayList<T>
```

Comment:
Sure:) I just thought that the logic in this class is easy enough to implement it directly on a plain C array.